api/src/main/java/datawave/webservice/model/Model.java

```diff
@@ -11,12 +11,17 @@
 import javax.xml.bind.annotation.XmlElementWrapper;
 import javax.xml.bind.annotation.XmlRootElement;
 
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.Multimap;
+
 import datawave.query.model.FieldMapping;
 import datawave.webservice.HtmlProvider;
 import datawave.webservice.result.BaseResponse;
-import lombok.Data;
 
-@Data
```

Comment:
Apparently I removed lombok...  thoughts?  opinions?

Reply:
Seems like it was unused, so probably fine. If we wanted to use lombok more places that would be fine too

Reply:
I have a mild preference for less code with lmbok, but there are valid arguments for and against.

Reply:
Let's minimize the scope of these changes.